CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Fixed
 - [#833](https://github.com/improbable-eng/thanos/issues/833) Store Gateway matcher regression for intersecting with empty posting.
 - [#867](https://github.com/improbable-eng/thanos/pull/867) Fixed race condition in sidecare between reloader and shipper.
+- [#910](https://github.com/improbable-eng/thanos/pull/910) Query's stores UI page is now sorted by type and old DNS or File SD stores are removed after 5 minutes (configurable via the new `--store.unhealthy-timeout=5m` flag).
```

Comment:
Isn't it rather "Added"?

Reply:
It's more of a enhancement :smile: updated